microshift_networking/microshift-kubeconfig.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="microshift-cluster-access-kubeconfig"]
+= Cluster access with kubeconfig
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-networking
+
+toc::[]
+
+Learn about how `kubeconfig` files are used with {product-title} deployments. These configuration files contain the necessary information about clusters, access type, IP addresses, and the certificates needed for authentication. CLI tools use `kubeconfig` files to communicate with the API server of a cluster.
```

Comment:
Not sure about the `access type`. It is more or less implied in the IP address/hostname mentioned right after.